core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/RequestManager.java

```diff
@@ -604,4 +495,28 @@
   public void addPoolWatcher(Pool pool, User watcher) throws IOException;
 
   public void removePoolWatcher(Pool pool, User watcher) throws IOException;
+
+  @Deprecated
+  long saveLibrary(Library library) throws IOException;
+
+  @Deprecated
+  long saveLibraryQC(LibraryQC libraryQc) throws IOException;
+
+  @Deprecated
+  public LibraryType getLibraryTypeByDescriptionAndPlatform(String description, PlatformType platformType) throws IOException;
+
+  @Deprecated
+  public LibrarySelectionType getLibrarySelectionTypeByName(String name) throws IOException;
+
+  @Deprecated
+  public LibraryStrategyType getLibraryStrategyTypeByName(String name) throws IOException;
+
+  @Deprecated
+  public long saveLibraryDilution(LibraryDilution libraryDilution) throws IOException;
+
+  @Deprecated
+  public Collection<Library> listAllLibrariesBySampleId(long sampleId) throws IOException;
+
+  @Deprecated
+  public Collection<LibraryDilution> listAllLibraryDilutions() throws IOException;
```

Comment:
Because the `FormUtils` class uses them, and its methods take a RequestManager parameter :((

I had assumed that also passing in the necessary services would not be desirable, but I didn't look into it beyond that. Do you have ideas as to how we might refactor that class?